addons/crm/models/crm_lead.py

```diff
@@ -1722,7 +1722,7 @@ def _prepare_customer_values(self, partner_name, is_company=False, parent_id=Fal
             'is_company': is_company,
             'type': 'contact'
         }
-        if self.lang_id:
+        if self.lang_id and self.lang_id.active:
```

Comment:
Could be simplified directly to
```suggestion
        if self.lang_id.active:
```